packages/shylu/shylu_dd/frosch/src/SchwarzPreconditioners/FROSch_OneLevelPreconditioner_def.hpp

```diff
@@ -52,14 +52,25 @@ namespace FROSch {
     SchwarzPreconditioner<SC,LO,GO,NO> (parameterList,k->getRangeMap()->getComm()),
     K_ (k),
     SumOperator_ (new SumOperator<SC,LO,GO,NO>(k->getRangeMap()->getComm())),
-    OverlappingOperator_ ()
+    MultiplicativeOperator_ (new MultiplicativeOperator<SC,LO,GO,NO>(k,parameterList)),
```

Comment:
Can we introduce an abstract class from which both, the SumOperator and the MultiplicativeOperator, are derived? Then, we just need to have operator here.

Reply:
Added to ToDo-List